src/utils/index.ts

```diff
@@ -167,7 +167,9 @@ export function noop() {}
 
 export const isIOSMobile = /iPad|iPhone|iPod/.test(navigator.userAgent);
 export const isAndroidMobile = /Android/.test(navigator.userAgent);
-export const isMobile = isIOSMobile || isAndroidMobile;
+export const isMobile = (fixateView?: 'mobile' | 'desktop') =>
+  fixateView === 'mobile' ||
+  (fixateView !== 'desktop' && (isIOSMobile || isAndroidMobile));
```

Comment:
Is this what you intended?

```suggestion
export const isMobile = (fixateView?: 'mobile' | 'desktop') => (fixateView === 'mobile' || isIOSMobile || isAndroidMobile);
```

Reply:
No. the whole point is to fixate to given value: either mobile or desktop. If desktop is given, we don't care about userAgent values